src/Controls/src/Core/Platform/AlertManager/AlertManager.Android.cs

```diff
@@ -285,35 +280,50 @@ void OnPromptRequested(IView sender, PromptArguments arguments)
 
 			void UpdateProgressBarVisibility(bool isBusy)
```

Comment:
The SetProgressBarIndeterminateVisibility method is deprecated and was not working. Replaced by a custom logic creating a ProgressBar on the fly.